action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Your name here'
```

Comment:
Do we need this file ?

Reply:
Yes. That will be that entry point for this action.